test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -18,18 +76,37 @@ namespace Test::KozaniManagerRuntimeTests
         TEST_CLASS_SETUP(ClassSetup)
         {
             ::TB::Setup();
+            ::TP::AddPackage(KozaniManagerTestPackage::c_PackageDirName, KozaniManagerTestPackage::c_PackageFullName);
             return true;
         }
 
         TEST_CLASS_CLEANUP(ClassCleanup)
         {
+            ::TP::RemovePackageIfNecessary(KozaniManagerTestPackage::c_PackageFullName);
             ::TB::Cleanup();
             return true;
         }
 
-        // TODO Replace this with your own tests
-        TEST_METHOD(JustDoIt)
+        TEST_METHOD(ActivateRemoteApplicationTests)
```

Comment:
Tests? Plural?

1 test should do 1 thing. If there's multiple tests please make multiple methods. Easier to debug, repro, yadda yadda.

Plus a 'Test*' suffix isn't needed since the method's in a BlahBlahTests class.

This has 1 test so SUGGESTION: Rename to `ActivateRemoteApplication_Launch`

Reply:
renamed.